common/v2/libs/formatters.ts

```diff
@@ -1,7 +1,27 @@
+import { toTokenBase } from './units';
+
 export function stripHexPrefix(value: string) {
   return value.replace('0x', '');
 }
 
 export function stripHexPrefixAndLower(value: string): string {
   return stripHexPrefix(value).toLowerCase();
 }
+
+export const buildEIP681EtherRequest = (
```

Comment:
move to `v2/services/EthService/utils`

Reply:
That directory doesn't exist in this PR yet